site/src/docs/components/header/index.mdx

```diff
@@ -22,56 +22,232 @@ export default ({ children, pageContext }) => <TabsLayout pageContext={pageConte
+              <Header.NavigationLink href="#" label="Deep link 2" />,
+            ]}
+          />,
+        ]}
+      />
+      <Header.NavigationLink
+        href="#"
+        label="Link 2"
+        dropdownLinks={[
+          <Header.NavigationLink
+            href="#"
+            label="Nested link 1"
+            active
+            dropdownLinks={[
+              <Header.NavigationLink href="#" label="Deep link 1" />,
+              <Header.NavigationLink href="#" label="Deep link 2" />,
+            ]}
+          />,
+          <Header.NavigationLink
+            href="#"
+            label="Nested link 2"
+            dropdownLinks={[
+              <Header.NavigationLink href="#" label="Deep link 1" />,
+              <Header.NavigationLink href="#" label="Deep link 2" />,
+            ]}
+          />,
+        ]}
+      />
+      <Header.NavigationLink href="#" label="Link 3" />
+    </Header.NavigationMenu>
+  </Header>
+</PlaygroundPreview>
 
-#### With navigation bar
+#### With universal bar
 
-Coming soon!
+- Using the Header.UniversalBar component is optional. Use it to provide frequently visited and useful links.
+- If the primary link information is provided, it should direct to the webpage of City of Helsinki.
+- Keep the link labels clear and concise. Prefer max 1-2 word long labels so the labels don't go to the next line on smaller screens. Avoid starting multiple link labels with the same word.
+- On smaller screens (768px and down) the Header.UniversalBar is hidden and the contents are moved inside the [the Header.ActionBar component's](#with-action-bar) menu.
+
+<PlaygroundPreview>
+  <Header>
+    <Header.UniversalBar primaryLinkHref="#" primaryLinkText="Helsingin kaupunki">
+      <Header.NavigationLink href="#" label="F.A.Q" />
+      <Header.NavigationLink href="#" label="User guide" />
+    </Header.UniversalBar>
+    <Header.ActionBar
+      title="Helsingin kaupunki"
+      titleHref="https://hel.fi"
+      titleHref="https://hel.fi"
+      logoAriaLabel="Service logo"
+      logoHref="https://hel.fi"
+      menuButtonAriaLabel="Menu"
+    />
+  </Header>
+</PlaygroundPreview>
 
 #### With navigation language selector
 
 - Using the Header.NavigationLanguageSelector is optional. Use the navigation language selector to define site languages. 
-- Header has <code>onDidChangeLanguage</code> function, where user can define a callback function which gets the selected language key (value) as parameter.
+- The Header component has `onDidChangeLanguage` prop function, where user can define a callback function which gets the selected language key (value) as parameter.
 - See example for Header.NavigationLanguageSelector <Link openInNewTab openInNewTabAriaLabel="Opens in a new tab" href="/storybook/react/?path=/story/components-header--minimal-with-localization">in Storybook</Link>
+- On smaller screens (768px and down) the Header.NavigationLanguageSelector moves under the Header.ActionBar.
 
 #### With skip link
 
```

Comment:
Use the skip link component to help keyboard-only users skip to the main content on a page.